bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -1,15 +1,28 @@
-Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function (instance, sandbox) {
-    this.instance = instance;
+Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function (locale, sandbox) {
+    this.locale = locale;
     this.sb = sandbox;
     this.service = sandbox.getService('Oskari.statistics.statsgrid.StatisticsService');
     this.spinner = Oskari.clazz.create('Oskari.userinterface.component.ProgressSpinner');
+    this.paramHandler = Oskari.clazz.create( 'Oskari.statistics.statsgrid.IndicatorParameterHandler', this.service, this.locale );
     this._values = {};
     this._selections = [];
+    this.parentElement = null;
     Oskari.makeObservable(this);
+    var me = this;
+    var errorService = this.service.getErrorService();
+
+    this.paramHandler.on('Data.Loaded', function ( data ) {
+        me.spinner.stop();
+        if ( Object.keys(data.selectors).length === 0 ) {
+            errorService.show(locale.erros.title, locale.errors.regionsetsIsEmpty);
```

Comment:
Based on the localization key the error messages says something about regionsets, but you are checking if the indicator had any selectors. Showing wrong error message here?